index.html

```diff
@@ -369,11 +167,14 @@ <h2>About me</h2>
                             <div class="contactFormSec_para">   
                                  <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>   
                             </div>
-                            <form action="https://formspree.io/f/xyyangae" method="post">    
-                                <input type="text" class="text-field" placeholder="Full name" minlength="1" maxlength="30" name="Name" required>
-                                <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+                            <form action="https://formspree.io/f/xyyangae" method="post" id="main_form">    
+                                <input type="text" class="text-field" placeholder="Full name" minlength="1" maxlength="30" name="Name" required id="name_text">
+                                <input type="email" class="text-field" placeholder="Email address" name="Email address" required id="email_text">
                                 
                                 <textarea name="short-writeup" id="bio" class="text-field-area" cols="30" rows="5" minlength="1" maxlength="500" placeholder="Write me something..." required></textarea>
+
+                                <small class="message"></small>
```

Comment:
- [ ] [OPTIONAL] Kindly consider using a shouting color like red. This color is mostly used to show the error messages and the user can easily be able to see the error. 
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233597106-596fdab3-1d90-48f4-ae30-0c13c2ad9f1f.png)